extensions/src/tables/index.ts

```diff
@@ -166,6 +198,51 @@ export default class Tables extends Extension<Details, Blocks> {
             newRow.push(0);
           }
           this.tables[table].push(newRow);
+          this.rowNames[table].push('row');
+        }
+      }),
+      // removes the specified column from the table
+      deleteColumn: (self: Tables) => ({
+        type: BlockType.Command,
+        args: [self.tableNamesArg, ArgumentType.Number],
```

Comment:
Since columns index from 1 (correct?) I think it be good to give these arguments a default value of `1` (instead of `0`)